File: TeamCodeC/src/main/java/org/firstinspires/ftc/teamcodec/config/OuttakeConstants.java
package org.firstinspires.ftc.teamcodec.config;

import com.bylazar.configurables.annotations.Configurable;

@Configurable
public class OuttakeConstants {
    public static double kp = 300;
    public static double ki = 0;
    public static double kd = 0;
    public static double ks = 0;
    public static double kv = 0;

    public static double defaultTargetVelocity = 1500;
    public static double allowance = 50;
}

File: TeamCodeD/src/main/java/org/firstinspires/ftc/teamcode/constants/OuttakeConstants.kt
package org.firstinspires.ftc.teamcode.constants

import com.bylazar.configurables.annotations.Configurable

@Configurable
object OuttakeConstants {
    @JvmField
    var KP: Double = 300.0

    @JvmField
    var KI: Double = 0.0

    @JvmField
    var KD: Double = 0.0

    @JvmField
    var KS: Double = 0.0

    @JvmField
    var DEFAULT_TARGET_VELOCITY: Double = 1500.0

    @JvmField
    var ALLOWANCE: Double = 50.0
}